tensorboard/plugins/core/core_plugin_test.py

```diff
@@ -162,6 +170,8 @@ def testDataPaths_disableAllCaching(self):
             self.assertEqual(200, response.status_code, msg=path)
             self.assertEqual("0", response.headers.get("Expires"), msg=path)
 
+    @unittest.skipIf(not ENABLE_DB_TESTS,
```

Comment:
nit: Prefer `@unittest.skipUnless(ENABLE_DB_TESTS, …)` for clarity.

Reply:
Done, thanks for the suggestion.